### ISSUE TITLE
Remote Inspector doesn't update when a Resource type is set in code

### ISSUE THREAD

**Issue by TimurBahadir** [godotengine/godot#28353]
**Godot version:**
3.1.1 RC1

**OS/device including version:**
Windows 10

**Issue description:**
When creating and assigning a CollisionShape in code, the remote inspector
says it is [empty] but it is actually assigned and working.

**Steps to reproduce:**
- Create Area2D node
- Create CollisionShape2D Node as child
- Add script to Area2D
```
  public override void _Ready()
  {
    CollisionShape2D collisionShape = GetChild(0) as CollisionShape2D;

    collisionShape.Shape = new RectangleShape2D();
  }
```
- Optionally enable Visible Collision Shapes

![Code_2019-04-23_22-05-24](https://user-images.githubusercontent.com/3100680/56612359-759b4580-6614-11e9-8e18-373cc2772e7b.png)

**Minimal reproduction project:**
[Issue.zip](https://github.com/godotengine/godot/files/3109399/Issue.zip)

**Comment by amisner2k** [comment]
Yes! This issue was happening for me and I thought I was going crazy. I'm glad it's been reported.

**Comment by KoBeWi** [comment]
Still valid in b7dc08f, even though a similar #30033 was fixed.

**Comment by timothyqiu** [comment]
Just encountered the same issue when debugging.

This is still valid in 3.2.3 / 3.2.4.beta4 / master(8ad0ff8ae).

**Comment by Zireael07** [comment]
Can we have this converted to a tracker? It seems many things don't show up in remote inspector...

**Comment by Zireael07** [comment]
Then maybe edit the issue title to mention Resources?

**Comment by akien-mga** [comment]
Fixed by #44657.
